python/py-aiodns/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-aiodns
+version             1.1.1
+categories-append   devel
+platforms           darwin
+license             MIT
+
+python.versions     27 34 35 36 37
```

Comment:
Unless there is a specific need for them, remove the older python versions py34 and py35.